common/app/common/utils/PriceFormatterImpl.java

```diff
@@ -13,7 +19,13 @@
     }
 
     @Override
-    public String format(MonetaryAmount price) {
-        return MonetaryFormats.getAmountFormat(locale).format(price);
+    public String format(MonetaryAmount monetaryAmount) {
+        final AmountFormatQuery amountFormatQuery = AmountFormatQueryBuilder.of(locale)
+                .set("pattern", "###0.00### ¤")
+                .set(CurrencyStyle.SYMBOL)
+                .build();
+        final MonetaryAmountFormat amountFormat = MonetaryFormats.getAmountFormat(amountFormatQuery);
+        return amountFormat.format(monetaryAmount)
+                .replace("EUR", "€");//yep, the money lib does not know the euro sign
```

Comment:
You can use the same I implemented in dmexco version, that one was implemented and tested:
https://github.com/sphereio/sphere-sunrise-dmexco/blob/master/common/app/common/utils/PriceFormatterImpl.java